setup.cfg

```diff
@@ -32,3 +32,9 @@ filterwarnings=
 
 [coverage:run]
 source_pkgs = starlette, tests
+
+[coverage:report]
+exclude_lines =
+    pragma: no cover
+    pragma: nocover
```

Comment:
Those 2 need to be included? :thinking:

Reply:
yes, overriding `exclude_lines` takes full control over what's excluded https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html.